src/pyscipopt/scip.pxi

```diff
@@ -6809,6 +6809,191 @@ cdef class Model:
 
         return pyCons
 
+
+    def addMatrixConsIndicator(
+        self,
+        cons: MatrixExprCons,
+        binvar: Union[Variable, MatrixVariable] = None,
+        activeone: Union[bool, np.ndarray] = True,
+        name: Union[str, np.ndarray] = "",
+        initial: Union[bool, np.ndarray] = True,
+        separate: Union[bool, np.ndarray] = True,
+        enforce: Union[bool, np.ndarray] = True,
+        check: Union[bool, np.ndarray] = True,
+        propagate: Union[bool, np.ndarray] = True,
+        local: Union[bool, np.ndarray] = False,
+        dynamic: Union[bool, np.ndarray] = False,
+        removable: Union[bool, np.ndarray] = False,
+        stickingatnode: Union[bool, np.ndarray] = False,
+    ):
+        """Add an indicator matrix constraint for the linear inequality `cons`.
+
+        The `binvar` argument models the redundancy of the linear constraint. A solution
+        for which `binvar` is 1 must satisfy the constraint.
+
+        Parameters
+        ----------
+        cons : MatrixExprCons
+            A linear inequality of the form "<=".
+
+        binvar : Variable or MatrixVariable, default None
+            Binary indicator variable / matrix variable, or None if it should be created.
```

Comment:
Differ from the PySCIPOpt document style, use the numpy doc style
- The first word uses uppercase.
- Two parameters add a blank line.
- Move the default value to the line of the parameter line.

Should I follow the PySCIPOpt document style?